### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by cormullion**
"mailing list" -> Discourse

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Hi! If you are new to the Julia community: welcome, and thanks for trying Julia.
 
 - Contributing code? Be sure to review the [contributor checklist](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#contributor-checklist) for helpful tips on the tools we use to build Julia.
 
-- Library feature requests are generally not accepted on this issue tracker. New libraries should be developed as [packages](http://docs.julialang.org/en/release-0.4/manual/packages/#package-development). Please email the [julia-users](https://groups.google.com/forum/#!forum/julia-users) mailing list to discuss library ideas. Doing so will often lead to pointers to existing projects and bring together collaborators with common interests.
+- Library feature requests are generally not accepted on this issue tracker. New libraries should be developed as [packages](http://docs.julialang.org/en/release-0.4/manual/packages/#package-development). Discuss ideas for libraries at [Julia Discourse forum](https://discourse.julialang.org/). Doing so will often lead to pointers to existing projects and bring together collaborators with common interests.
```
Comment (nalimilan) [inline CONTRIBUTING.md]: Not a native speaker, but maybe "the Julia Discourse forum"?

Comment (cormullion) [inline CONTRIBUTING.md]: Good idea!

**Review by nalimilan — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Hi! If you are new to the Julia community: welcome, and thanks for trying Julia.
 
 - Contributing code? Be sure to review the [contributor checklist](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#contributor-checklist) for helpful tips on the tools we use to build Julia.
 
-- Library feature requests are generally not accepted on this issue tracker. New libraries should be developed as [packages](http://docs.julialang.org/en/release-0.4/manual/packages/#package-development). Please email the [julia-users](https://groups.google.com/forum/#!forum/julia-users) mailing list to discuss library ideas. Doing so will often lead to pointers to existing projects and bring together collaborators with common interests.
+- Library feature requests are generally not accepted on this issue tracker. New libraries should be developed as [packages](http://docs.julialang.org/en/release-0.4/manual/packages/#package-development). Discuss ideas for libraries at the [Julia Discourse forum](https://discourse.julialang.org/). Doing so will often lead to pointers to existing projects and bring together collaborators with common interests.
```
Comment (vchuravy) [inline CONTRIBUTING.md]: Also not a native speaker, but I would write "on the"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -240,7 +240,7 @@ Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request w
 
 * Julia
   - **Homepage:** <http://julialang.org>
-  - **Mailing lists:** <http://julialang.org/community/>
+  - **Discussion:** <http://julialang.org/community/>
```
Comment (vchuravy) [inline CONTRIBUTING.md]: Maybe simply call it *Community*, since neither `Mailing lists` nor `Discussion` is quite right

**Review by vchuravy — APPROVED** [review]
Looks good to me. Thank you for doing this.

File: CONTRIBUTING.md (hunk 3 above)
Comment (cormullion) [inline CONTRIBUTING.md]: Yup, good idea.

File: CONTRIBUTING.md (hunk 2 above)
Comment (kshyatt) [inline CONTRIBUTING.md]: I think "at" is fine here

**Comment by cormullion** [comment]
Thanks!

**Pull request merged**